warehouse/query-core/src/main/java/datawave/query/jexl/lookups/LookupTermsFromRegex.java

```diff
@@ -37,48 +36,32 @@
 import com.google.common.collect.Iterators;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Multimap;
-import com.google.common.collect.Sets;
 
 public class LookupTermsFromRegex extends RegexIndexLookup {
     private static final Logger log = Logger.getLogger(LookupTermsFromRegex.class);
     
     protected MetadataHelper helperRef = null;
-    protected Set<String> datatypeFilter;
-    protected Set<Text> fields;
-    protected Set<Text> reversefields;
+    protected Set<String> fields;
+    protected Set<String> reverseFields;
     protected final Set<String> patterns;
+    protected boolean unfieldedLookup = false;
     
-    public LookupTermsFromRegex(Set<String> fields, Set<String> patterns, MetadataHelper helperRef) {
+    public LookupTermsFromRegex(Set<String> fields, Set<String> reverseFields, Set<String> patterns, MetadataHelper helperRef, boolean unfieldedLookup) {
         setMetadataHelper(helperRef);
         this.patterns = patterns;
-        setFields(fields);
-        this.datatypeFilter = Sets.newHashSet();
+        this.fields = fields;
+        this.reverseFields = fields;
```

Comment:
probably want to assign `reverseFields` to `this.reverseFields` here.

Reply:
another great find @drewfarris .  I will also work to add some test cases as we obviously have some holes in that regard.